modules/virt-creating-configmap.adoc

```diff
@@ -1,16 +1,9 @@
 // Module included in the following assemblies:
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
```

Comment:
Less important than the subsequent comment but line 2 should be a commented new line. Ie: 
```
// Module included in the following assemblies:
// 
// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
```

Reply:
Less important? Definitely.